src/index.html

```diff
@@ -16,17 +16,29 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body>
+    <form
+      class="searchbar"
+      data-qa="big"
+    >
+      <input
+        class="searchbar__input searchbar__input-big"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
+    <form
+      data-qa="small"
```

Comment:
The `data-qa` attribute is used here with the value 'small'. Ensure that this value is unique and descriptive enough to differentiate it from other similar attributes.